diff --git a/Panel.qml b/Panel.qml
index af654fa..444991f 100644
--- a/Panel.qml
+++ b/Panel.qml
@@ -1,4 +1,5 @@
 import QtQuick
+import "../../ui/panel-shadows" as ShadowPanels
 import QtQuick.Controls
 import qs.Commons
 import qs.Ui
@@ -613,7 +614,7 @@ Panel {
     onPressed: function(buttonCode) { root.toggle() }
   }
 
-  KeyboardPanel {
+  ShadowPanels.KeyboardPanel {
     id: panel
     anchorItem: button
     owner: root
